### DISCUSSION TITLE
Huawei Cloud connectors - Open discussion

### DISCUSSION THREAD

**Discussion by davira** [airbytehq/airbyte#36118] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Huawei Cloud is the top 5 largest CSP in the global market, but Airbyte has no native connection to Huawei Cloud services. 
Source and destinations include (but are not limited to):

- Data Warehouse Service
- DLI (Data Lake Inisight)
- OBS (Object Storage)
- DIS (Data Ingestion Service, similar to Kafka)

* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 
Support HWC data services (e.g. RDS, Data Warehouse Service, DLI, OBS (object storage)).

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
Many Huawei Cloud customer wish to use Airbyte for moving data, but cannot due to the lack of native connectors. Till now, Airbyte can only provide Postgres connector to DWS (not optimized)

* How often do you want to run syncs?
Customers run the syncs from CDC, to hourly, daily, weekly, monthly

* If this is an API source connector, which entities/endpoints do you need supported?
HWC provides APIs or JDBC/ODBC connectors for different cloud services

* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? 
Not able to answer now

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
HWC provides our own tools for moving data, such as **CDM**. We also provide jdbc/odbc, apis, etc. 
But, we do belive on the open source community and ecosystem, therefore, we want to survey available market tools

## Are you willing to submit a PR?
As done with other open source projects, I can request internal resources (from developers, cloud resources for development environment, etc), with the condition that the connectos can be accepted by the Airbyte community (pending to discuss in details)

I appreciate the Airbyte community time and support.

**Comment by misteryeo** [comment]
Thanks for reaching out to us @davira, let us take a look at what you've shared and we'll come back to you soon!

**Comment by davira** [comment]
Thanks @misteryeo ! Let me know any advancements

[a comment by an automated account is not shown]
